packages/components/src/external-link/index.tsx

```diff
@@ -65,18 +63,16 @@ function UnforwardedExternalLink(
 			target="_blank"
 			rel={ optimizedRel }
 			ref={ ref }
+			style={ { textDecoration: 'none' } }
 		>
-			{ children }
+			<span style={ { textDecoration: 'underline' } }>{ children }</span>
 			<VisuallyHidden as="span">
 				{
 					/* translators: accessibility text */
 					__( '(opens in a new tab)' )
 				}
 			</VisuallyHidden>
-			<StyledIcon
-				icon={ external }
-				className="components-external-link__icon"
-			/>
+			&nbsp;&#8599;
```

Comment:
Any concerns with font browser support? Are we certain that the user won't use a font that doesn't support this unicode arrow icon?

Reply:
It's possible, but in such cases I believe a fallback font would be used to display the correct symbol, with a slightly different appearance, e.g.:

<img width="275" alt="Screenshot 2024-03-28 at 16 56 43" src="https://github.com/WordPress/gutenberg/assets/846565/d21b6dee-082e-4aea-86c2-eeec34f3dbba">

Reply:
Sounds good!